src/test/mediapipe/relative_paths/config_relative_dummy_subconfig.json

```diff
@@ -4,12 +4,12 @@
         {
             "name":"mediapipeAddADAPTFULL",
             "graph_path":"graph1/graphaddadapterfull.pbtxt",
-            "subconfig":"graph1/config.json"
+            "subconfig":"graph1/subconfig.json"
```

Comment:
Please rename this config json - it uses model add instead of dummy so we should be consistent here.